lighthouse-core/gather/gatherers/js-usage.js

```diff
@@ -11,6 +11,22 @@ const Gatherer = require('./gatherer.js');
  * @fileoverview Tracks unused JavaScript
  */
 class JsUsage extends Gatherer {
+  constructor() {
+    super();
+    /** @type {LH.Crdp.Debugger.ScriptParsedEvent[]} */
+    this._scriptParsedEvents = [];
+    this.onScriptParsed = this.onScriptParsed.bind(this);
+  }
+
+  /**
+   * @param {LH.Crdp.Debugger.ScriptParsedEvent} event
+   */
+  onScriptParsed(event) {
```

Comment:
you're missing a 

`    driver.on('Debugger.scriptParsed', this.onScriptParsed);`

Reply:
thanks! I was just debugging why my test wasn't work and this comment was a blessing 🙏